index.src.html

```diff
@@ -569,8 +569,8 @@ <h3 id="referrer-policy-header">Delivery via Referrer-Policy header</h3>
 
   The syntax for the name and value of the header are described by the following
   ABNF grammar. ABNF is defined in [[!RFC5234]], and the <code>#rule</code> ABNF
-  extension used below is defined in <a
-  href="https://www.rfc-editor.org/rfc/rfc9110#section-5.6.1">Section 5.6.1</a> of [[!RFC9110]].
+  extension used below is defined in [[RFC9110#abnf.extension|Section 5.6.1]] of
```

Comment:
This link does not show up in the preview https://pr-preview.s3.amazonaws.com/tidoust/webappsec-referrer-policy/pull/169.html#referrer-policy-header

Reply:
This seems to be the cause of the build error too.

Reply:
Yes, see my previous comment: https://github.com/w3c/webappsec-referrer-policy/pull/169#issuecomment-1825858988

The new data has now reached the online Bikeshed service. I re-ran the CI build, which now passes. I don't think I can run PR preview again though.